app.js

```diff
@@ -176,6 +176,7 @@ app.post('/api/upload', upload.single('myFile'), apiController.postFileUpload);
 app.get('/api/pinterest', passportConfig.isAuthenticated, passportConfig.isAuthorized, apiController.getPinterest);
 app.post('/api/pinterest', passportConfig.isAuthenticated, passportConfig.isAuthorized, apiController.postPinterest);
 app.get('/api/google-maps', apiController.getGoogleMaps);
+app.get('/api/google-drive', apiController.getGoogleDrive);
```

Comment:
Since accessing the API requires authentication, you would need to check if the user is authenticated and authorized.  Otherwise, if a user that hasn't logged in goes to the /api/google-drive then the server will give a 500 error with 
```
TypeError: Cannot read property 'accessToken' of undefined
    at exports.getGoogleDrive (/mnt/c/temp/g/hackathon-starter/controllers/api.js:714:25)
```
Simply change of the above line to:
`app.get('/api/google-drive', passportConfig.isAuthenticated, passportConfig.isAuthorized, apiController.getGoogleDrive);`

which calls to https://github.com/sahat/hackathon-starter/blob/master/config/passport.js#L590 and https://github.com/sahat/hackathon-starter/blob/master/config/passport.js#L600 to check to see if the user is logged in and if we already have a token for the API and if not to obtain one.

Reply:
By the way, the Auth path will need to be handled as well since the current isAuthorized will try `/auth/google-drive` which is not defined.  I haven't had a chance to look, but if a separate auth path can be set for the google-drive then perhaps that path can "add" the extra drive permission, while the other google auth path doesn't.  Another option that I can think of would be to rename this example to 'google api example' (/api/google) and have this PR start that example with a google drive example in the view.  In this way, isAuthorize will default to `/auth/google` which is already defined.

Reply:
What do you think about using 'google' as prefix for URLs like

'/google/drive'
'/google/analytics'